ismatec/driver.py

```diff
@@ -163,18 +163,18 @@ async def get_setpoint_type(self, channel: int) -> Protocol.Setpoint:
         return Protocol.Setpoint(type_code)
 
     async def set_setpoint_type(self, channel: int, type: Protocol.Setpoint):
-        """Set the setpoint type (RPM or flowrate) on the specified channel."""
+        """Set the setpoint type (RPM or flow rate) on the specified channel."""
```

Comment:
I made all the docstrings follow the same writing pattern. I find it subtly easier to understand a new driver when all the docstrings have the same exact grammar.